CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 * [FEATURE] Call RUM's `errorEventMapper` for crashes. See [#1742][]
 * [FEATURE] Support calling log event mapper for crashes. See [#1741][]
 * [FIX] Fix crash in `NetworkInstrumentationFeature`. See [#1767][]
+* [FEATURE] Add support for 128 bit trace IDs. See [#1721][]
+
+[#1759]:
```

Comment:
blocking/ There's something wrong with this one

Reply:
there is a reason one must review first their own PRs :)